### ISSUE TITLE
Bootstrapping a project in Git Bash under Windows generates wrong @Path

### ISSUE THREAD

**Issue by axblu** [quarkusio/quarkus#17939]
## Describe the bug
When bootstrapping a Quarkus app in Git Bash under Windows a wrong `@Path` is inserted.
Instead of `@Path("/hello")` it is `@Path("/C:/Dev/Tools/Git/hello")`. The same is true for the generated test class.

### Expected behavior
The path is `/hello`.

### Actual behavior
The path is `/C:/Dev/Tools/Git/hello` (depending on the installation folder of your Git Bash).

## To Reproduce
1. Open Git Bash
2. Run either command for Windows from the [Getting Started Guide](https://quarkus.io/guides/getting-started#bootstrapping-the-project):
`mvn io.quarkus:quarkus-maven-plugin:1.13.7.Final:create -DprojectGroupId=org.acme -DprojectArtifactId=getting-started -DclassName="org.acme.getting.started.GreetingResource" -Dpath="/hello"`
or
`mvn io.quarkus:quarkus-maven-plugin:1.13.7.Final:create "-DprojectGroupId=org.acme" "-DprojectArtifactId=getting-started" "-DclassName=org.acme.getting.started.GreetingResource" "-Dpath=/hello"`
3. Check `GreetingResource.java`

### Screenshots
![getting-startet-git-bash](https://user-images.githubusercontent.com/72867162/122171207-c1a64580-ce7f-11eb-98d5-64eb24bc11a2.jpg)
![greeting-started-git-bash-test](https://user-images.githubusercontent.com/72867162/122171220-c4a13600-ce7f-11eb-88a4-e657a961d14b.jpg)

## Environment
Windows
Git for Windows 2.27
Java 8
Maven 3.6.3

**Comment by famod** [comment]
Ha, this is one of the first problems I ran into back then when I first tried out Quarkus and it's actually not a Quarkus bug: https://github.com/quarkusio/quarkus/issues/2149#issuecomment-553611171

TL;DR: Try `-Dpath=hello` (without slash) to avoid triggering that MinGW path conervsion.
